data/visibility.txt

```diff
@@ -970,10 +974,13 @@ world  00000000000000000000  +
         multi  00000000000000000000  -
         scuba_diving  00000000000000000000  -
         shooting  00000000000000000000  -
+        skateboard  00000000000000000000  -
```

Comment:
@vng do we need this file if it's completely ignored now?

Reply:
No we don't.

Reply:
@j13m126 No need any actions from your side.